app/Models/Traits/MarkdownHtmlTrait.php

```diff
@@ -22,11 +22,12 @@ protected function cleanAndSetMarkdown(string $name, ?string $markdown)
     }
 
     /**
+     * TODO: remove
      * Return HTML for body.
      *
      * @return string|null
      */
-    protected function getBodyHtmlAttribute() : ?string
+    protected function xgetBodyHtmlAttribute() : ?string
```

Comment:
Is this going to be removed in a following PR?

Reply:
Yes.

Reply:
Resolved in 0b25d4e9f9768151d8c77c5970204973e8ac7a0a